docs/user-interface/controls/graphicsview.md

```diff
@@ -0,0 +1,96 @@
+    <StackLayout>
+        <GraphicsView Drawable="{StaticResource drawable}"
+                      HeightRequest="500"
+                      WidthRequest="400" />
+    </StackLayout>
+</ContentPage>
+```
+
+## Position and size graphical objects
+
+The location and size of the `ICanvas` on a page can be determined by examining properties of the `RectangleF` argument in the `Draw` method.
+
+The `RectangleF` struct defines the following properties:
+
+- `Bottom`, of type `float`, which represents the y-coordinate of the bottom edge of the canvas.
+- `Center`, of type `PointF`. which specifies the coordinates of the center of the canvas.
+- `Height`, of type `float`, which defines the height of the canvas.
+- `IsEmpty`, of type `bool`, which indicates whether the canvas has a zero size and location.
+- `Left`, of type `float`, which represents the x-coordinate of the left edge of the canvas.
+- `Location`, of type `PointF`, which defines the coordinates of the upper-left corner of the canvas.
+- `Right`, of type `float`, which represents the x-coordinate of the right edge of the canvas.
+- `Size`, of type `SizeF`, which defines the width and height of the canvas.
+- `Top`, of type `float`, which represents the y-coordinate of the top edge of the canvas.
+- `Width`, of type `float`, which defines the width of the canvas.
+- `X`, of type `float`, which defines the x-coordinate of the upper-left corner of the canvas.
+- `Y`, of type `float`, which defines the y-coordinate of the upper-left corner of the canvas.
+
+These properties can be used to position and size graphical objects on the `ICanvas`. For example, graphical objects can be placed at the center of the `Canvas` by using the `Center.X` and `Center.Y` values as arguments to a drawing method. For information about drawing on an `ICanvas`, see [Draw graphical objects](~/user-interface/graphics/draw.md).
+
+## Convert the drawable to an image
+
+The graphical objects that are drawn on a `GraphicsView` can be converted to an image by the `ToImage` method, which is available in the `Microsoft.Maui.Graphics` namespace. This method requires `width` and `height` arguments, of type `float`, that specify the dimensions of the image.
+
+The `ToImage` method operates on an `IDrawable` object, which is exposed by the `GraphicsView.Drawable` property. Therefore, to call the `ToImage` method on a `GraphicsView`, the `GraphicsView` must be named with the `x:Name` attribute:
+
+```xaml
+<GraphicsView x:Name="graphicsDrawable"
+              Drawable="{StaticResource drawable}"
+              HeightRequest="500"
+              WidthRequest="400" />
+```
+
+In code, the `Drawable` property of the `GraphicsView` object can then be accessed, and the `ToImage` method called:
+
+```csharp
+using Microsoft.Maui.Graphics;
+...
+IImage image = graphicsDrawable.Drawable.ToImage(400, 500);
+```
+
+For information about image handling in `Microsoft.Maui.Graphics`, see [Images](~/user-interface/graphics/images.md).
```

Comment:
Could we include information about how to invalidate the Canvas?. https://github.com/dotnet/maui/pull/3142
I guess could be a typical question.